### PR TITLE
[INLONG-5066][Manager] Remove the LightGroup-related workflow listener and definitions

### LINKED ISSUE #5066: [Improve][Manager] Remove the LightGroup-related workflow listener and definitions
### Description

Remove the LightGroup-related workflow listener and definitions.

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by healchow**
### Prepare a Pull Request

- Fixes #5066

### Motivation

Remove the LightGroup-related workflow listener and definitions.

### Modifications

1. Merge `LightGroupXxxListener` into `InitGroupXxxListener` and `UpdateGroupXxxListener`.
2. Merge `XxxLightGroupWorkflowDefinition` into `XxxGroupWorkflowDefinition`.

### Verifying this change

- [x] This change is already covered by existing tests, such as:
   `InlongGroupProcessOperationTest`
   `CreateGroupWorkflowDefinitionTest`
   `WorkflowServiceImplTest`

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

**Comment by GanfengTan** [comment]
@healchow 
1.Please added background and benefits of modification.
2.Refactoring design needs to be added.

**Comment by healchow** [comment]
> @healchow 1. Please added background and benefits of modification. 2. Refactoring design needs to be added.

Added in the parent issue, see #5029.

**Comment by dockerzhang** [comment]
@woofyzhao @gong PTAL, thanks.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java [hunk 1]
```diff
@@ -129,11 +129,12 @@ public InlongGroupContext init() throws Exception {
...
-                formDataNew, NewGroupProcessForm.class);
-        Preconditions.checkNotNull(newGroupProcessForm, "NewGroupProcessForm cannot be null");
-        groupContext.setInitMsg(newGroupProcessForm);
-        WorkflowResult startWorkflowResult = workFlowClient.startInlongGroup(taskId, newGroupProcessForm);
+        ApplyGroupProcessForm groupProcessForm = JsonUtils.parseObject(
+                formDataNew, ApplyGroupProcessForm.class);
+        Preconditions.checkNotNull(groupProcessForm, "ApplyGroupProcessForm cannot be null");
+        groupContext.setInitMsg(groupProcessForm);
+        assert groupProcessForm != null;
+        WorkflowResult startWorkflowResult = workFlowClient.startInlongGroup(taskId, groupProcessForm);
```
Comment (woofyzhao) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:137]: I think it should be "workflowClient"

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/GroupCheckService.java [hunk 2]
```diff
@@ -37,30 +35,28 @@
...
                 String.format(ErrorCodeEnum.USER_IS_NOT_MANAGER.getMessage(), operator, managers));
 
-        GroupStatus state = GroupStatus.forCode(inlongGroupEntity.getStatus());
-        // Add/modify/delete is not allowed under certain group state
-        if (GroupStatus.notAllowedUpdate(state)) {
-            LOGGER.error("inlong group status was not allowed to add/update/delete related info");
-            throw new BusinessException(ErrorCodeEnum.OPT_NOT_ALLOWED_BY_STATUS);
+        GroupStatus status = GroupStatus.forCode(inlongGroupEntity.getStatus());
+        if (GroupStatus.notAllowedUpdate(status)) {
+            throw new BusinessException(String.format(ErrorCodeEnum.OPT_NOT_ALLOWED_BY_STATUS.getMessage(), status));
```
Comment (woofyzhao) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/group/GroupCheckService.java:59]: Does the string constant have place holder for status to format?

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java (hunk 1 above)
Comment (healchow) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:137]: Yeah, I think so.
But this class belongs to the client module and should change in another single PR.

Comment (woofyzhao) [reply inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:137]: private final WorkflowClient workFlowClient;
I was referring to this member declaration in the class you have modified.

**Review by woofyzhao — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/listener/InitGroupListener.java [hunk 3]
```diff
@@ -15,60 +15,57 @@
...
-        LightGroupResourceProcessForm form = (LightGroupResourceProcessForm) context.getProcessForm();
+    public ListenerResult listen(WorkflowContext context) throws WorkflowListenerException {
+        GroupResourceProcessForm form = (GroupResourceProcessForm) context.getProcessForm();
+        String groupId = form.getInlongGroupId();
+        log.info("begin to execute InitGroupListener for groupId={}", groupId);
+
         InlongGroupInfo groupInfo = form.getGroupInfo();
         if (groupInfo == null) {
-            throw new WorkflowListenerException(ErrorCodeEnum.GROUP_NOT_FOUND.getMessage());
+            throw new BusinessException("inlong group info cannot be null for init group process");
```
Comment (gong) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/listener/InitGroupListener.java:61]: why here change to `BusinessException` and throws `WorkflowListenerException`

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/CreateGroupWorkflowDefinition.java [hunk 4]
```diff
@@ -41,61 +41,61 @@
...
+
         // Start node
         StartEvent startEvent = new StartEvent();
         process.setStartEvent(startEvent);
 
-        // init MQ
+        // Init MQ
         ServiceTask initMQTask = new ServiceTask();
-        initMQTask.setName("initMQ");
+        initMQTask.setName("InitMQ");
```
Comment (kipshi) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/CreateGroupWorkflowDefinition.java:74]: make it enum

Comment (healchow) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/workflow/group/CreateGroupWorkflowDefinition.java:74]: Good idea.
I'll change them to Enum in the next PR.

**Review by gong — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Review by haibo-duan — APPROVED** [review]

**Review by GanfengTan — APPROVED** [review]

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/ServiceBaseTest.java [hunk 5]
```diff
@@ -17,28 +17,128 @@
...
 @SpringBootApplication
 @SpringBootTest(classes = ServiceBaseTest.class)
 public class ServiceBaseTest extends BaseTest {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceBaseTest.class);
-
     public static final String GLOBAL_GROUP_ID = "global_group";
     public static final String GLOBAL_STREAM_ID = "global_stream";
     public static final String GLOBAL_OPERATOR = "admin";
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceBaseTest.class);
```
Comment (kipshi) [inline inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/ServiceBaseTest.java:53]: why put here?

**Review by leosanqing — APPROVED** [review]

File: inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/ServiceBaseTest.java (hunk 5 above)
Comment (healchow) [reply inlong-manager/manager-service/src/test/java/org/apache/inlong/manager/service/ServiceBaseTest.java:53]: For reuse code.

**Pull request merged**